src/pages/App.tsx

```diff
@@ -1,98 +1,103 @@
 import React, { Suspense } from 'react'
-import { HashRouter, Route, Switch } from 'react-router-dom'
 import styled from 'styled-components'
 import GoogleAnalyticsReporter from '../components/analytics/GoogleAnalyticsReporter'
-import Header from '../components/Header'
 import Popups from '../components/Popups'
+import Background from '../components/Background'
+import { HashRouter, Switch, Route } from 'react-router-dom'
 import Web3ReactManager from '../components/Web3ReactManager'
-import DarkModeQueryParamReader from '../theme/DarkModeQueryParamReader'
 import AddLiquidity from './AddLiquidity'
 import {
   RedirectDuplicateTokenIds,
   RedirectOldAddLiquidityPathStructure,
   RedirectToAddLiquidity
 } from './AddLiquidity/redirects'
 import Pool from './Pool'
+import Farm from './Farm'
+import FarmSelect from './Farm/reselect'
+import Lending from './Lending'
+import Home from './Home'
 import PoolFinder from './PoolFinder'
 import RemoveLiquidity from './RemoveLiquidity'
 import { RedirectOldRemoveLiquidityPathStructure } from './RemoveLiquidity/redirects'
 import Swap from './Swap'
 import { RedirectPathToSwapOnly, RedirectToSwap } from './Swap/redirects'
-import { RightFlashIcon, LeftFlashIcon } from '../components/FlashIcon'
 import Bridge from './Bridge'
 import { RedirectToDefault } from './redirects'
-import Banner from '../components/Banner'
-import bannerData from '../static/banner.json'
+import Menu from '../components/Menu'
 
 const AppWrapper = styled.div`
   display: flex;
-  flex-flow: column;
-  align-items: flex-start;
-  overflow-x: hidden;
+  flex-direction: row;
+  max-height: 100vh;
+  align-items: stretch;
 `
-
-const HeaderWrapper = styled.div`
-  ${({ theme }) => theme.flexRowNoWrap}
-  width: 100%;
-  justify-content: space-between;
+const LeftMenu = styled.div`
+  height: 100vh;
+  flex: 0 0 19%;
+  order: -1;
+  ${({ theme }) => theme.mediaWidth.upToSmall`display: none;flex: 0 0 0`};
+  ${({ theme }) => theme.mediaWidth.upToMedium`display: none;flex: 0 0 22%`};
 `
 
-const BodyWrapper = styled.div`
-  display: flex;
-  flex-direction: column;
+const Sticky = styled.div`
+  position: fixed;
+  position: -webkit-fixed;
+  top: 0;
+  width: 19%;
+  height: 100vh;
+  ${({ theme }) => theme.mediaWidth.upToSmall`display: none;`};
+`
+const Content = styled.div`
+  background: black;
+  order: 1;
+  padding-top: 1rem;
   width: 100%;
-  padding-top: 160px;
-  align-items: center;
-  flex: 1;
-  overflow-y: auto;
-  overflow-x: hidden;
-  z-index: 10;
-
-  ${({ theme }) => theme.mediaWidth.upToExtraSmall`
-      padding: 16px;
-  `};
-
-  z-index: 1;
+  position: relative;
 `
 
-const Marginer = styled.div`
-  margin-top: 5rem;
+const Zindex = styled.div`
```

Comment:
why is this called zIndex

Reply:
yep. should be called `fullscreen` or something